src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -15,10 +15,25 @@ pr:
     - release/*
     - internal/release/*
 
-stages:
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - template: templates/stages/vmr-scan.yml
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-- template: /src/installer/eng/pipelines/templates/stages/vmr-build.yml
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
-    isBuiltFromVmr: true
+    pool:
+      name: NetCore1ESPool-Internal
+      image: 1es-windows-2022
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling

```

Comment:
I think these might be just copilot hallucinations as the 1ES tooling apparently uses it. I think the tag might not be needed

Reply:
Would be nice to get communication from 1es/ddfun/engsvcs on this.

Reply:
I did a bare minimum of searching this tag on the 1ES wiki and found nothing. Will ask around